payu/models/access.py

```diff
@@ -218,6 +219,12 @@ def setup(self):
                 f90nml.write(cpl_nml, nml_work_path + '~')
                 shutil.move(nml_work_path + '~', nml_work_path)
 
+            if model.model_type == 'cice':
+                # Set up and check the cice restart files.
+                cice4_make_restart_pointer(model,
+                                           run_start_date,
+                                           previous_runtime)
+
```

Comment:
I probably just didn't read the description carefully enough.

Can this be a function in the cice class, and it just's called here?

i.e. move cice4_make_restart_pointer & associated files to the Cice class. and then this line becomes

`model.cice4_make_restart_pointer(run_start_date,
                                           previous_runtime)`

If you do this, a better name might be `overwrite_restart_ptr()` ?

Reply:
That's a good idea, I think it helps to clean up the structure. I've added this in c1b3f57